src/foundation.cs

```diff
@@ -8151,7 +8151,7 @@ interface NSString2 : NSSecureCoding, NSMutableCopying, CKRecordValue
 		[iOS (10,0), Watch (3,0), NoTV, Mac (10,14, onlyOn64: true)]
```

Comment:
since it seems new on macOS you can avoid the compat/obsolete version for XM

Reply:
Ah thanks, good point I didn't realize that.